celery/backends/redis.py

```diff
@@ -141,10 +146,6 @@ def __init__(self, host=None, port=None, db=None, password=None,
             self.connparams.update(ssl)
             self.connparams['connection_class'] = redis.SSLConnection
 
-        if url:
```

Comment:
From what I have tested with `_params_from_url` no replacement will occur here, because the SSL options are passed as the `defaults` dictionary. Would you mind explaining why this is required?